web/src/components/feature/FeedFlowInput.tsx

```diff
@@ -1,40 +1,72 @@
 import { Switch, TextField } from '@equinor/eds-core-react'
 import styled from 'styled-components'
-import { TFeedFlow } from '../../types'
+import { molePerStandardCubicMeter } from '../../constants'
+import { useState } from 'react'
 
 const FlexContainer = styled.div`
   display: flex;
   gap: 16px;
   max-width: 250px;
 `
 
+function convertCubicFeedFlowValue(
```

Comment:
I know I send in cubicFeedFlow. The question is what I get out. Perhaps rename to `getMassFeedFlow` or something like that, and similarly for the function below

Reply:
Yeah. Was trying to avoid a very long name, but probably better clarity that way.